target/linux/ath79/image/mikrotik.mk

```diff
@@ -1,5 +1,17 @@
 include ./common-mikrotik.mk
 
+# N.B. kernel2minor is "kernel-to-mikrotik-nor" that packs the kernel to yaffs2 for routerboot
```

Comment:
consider removing this comment, or is there something special with this device about it?

Reply:
I can take it out. (I had it there as it took me a while to work out what it even was).

Reply:
BTW, should I "resolve" the conversations on these outdated files on this pull request?

Reply:
Depends. If I just say "do A", and you did A, then there is no reason to keep it open.
If you solved it differently, or did not solve it at all because you had a different opinion, closing the conversation might look like you want to hide something.

In your case it should all be boring and can be resolved.

Reply:
Understood, thanks.

Also what else do I need to do to advance this PR?

Reply:
I only looked at some parts of this. Someone will have to review the whole thing, especially the drivers ...